### ISSUE TITLE
Ability to load test at 1080p

### ISSUE THREAD

**Issue by jugory** [livekit/livekit-cli#103]
It would be great for my use case to test publishing 1080p video using the load-test command.

Proposing a `very-high` option be added to serve this use case

**Comment by jugory** [comment]
It does appear to already. I can only get this reliably but using the `--no-simulcast` argument

**Comment by davidzhao** [comment]
I don't believe we can do 1080p right now, mainly because we don't have a 1080p [video resource](https://github.com/livekit/livekit-cli/tree/main/pkg/provider/resources) to playback. It's a good idea to add one though!
